packages/output/src/utils/styles.tsx

```diff
@@ -152,6 +152,15 @@ function CustomStyles() {
                 margin-bottom: 16px;
                 text-align: center;
               }
+
+              amp-story-audio-sticker {
+                height: 100%;
+              }
+
+              amp-story-grid-layer div:has(~ .small) > span,
+              amp-story-grid-layer div:has(~ .large) > span {
+                font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;
+              }
```

Comment:
This does not appear to be working. Probably because our AMP sanitizer strips this as it doesn't know where it is being used, so it thinks it's unused & removes it.

I think we can solve this by wrapping the `<amp-story-audio-sticker>` in the `AudioStickerOutput` component in an extra `<div className="audio-sticker">`.

Then, here we can do `.audio-sticker { font-family: ... }`

Please test this.

Reply:
Yes, tested this and it worked! Pushed commits for the same.